### PR TITLE
Oci image tag feature

### PR DESCRIPTION
**Pull request by shimish2**

### CODE REVIEW

File: artifact/image/image.go [hunk 1]
```diff
@@ -60,12 +60,13 @@ func (a Artifact) Inspect(ctx context.Context) (types.ArtifactReference, error)
...
 func (a Artifact) inspect(ctx context.Context, imageID string, missingImage bool, diffIDs []string) error {
 	done := make(chan struct{})
-	errCh := make(chan error)
+	errCh := make(chan error, len(diffIDs))
 
 	var osFound types.OS
 	for _, d := range diffIDs {
 		go func(diffID string) {
 			layerInfo, err := a.inspectLayer(diffID)
+
```
Comment (rchincha) [inline artifact/image/image.go]: remove this whitespace line

**Comment by rchincha** [comment]
@shimish2
Added Support for OCI Image Tag

change this to say "add support for individual image tags"

we may want to upstream this someday.

**Comment by shimish2** [comment]
> @shimish2
> Added Support for OCI Image Tag
> 
> change this to say "add support for individual image tags"
> 
> we may want to upstream this someday.

Done

File: artifact/image/image.go (hunk 1 above)
Comment (shimish2) [reply artifact/image/image.go]: done

**Review by rchincha — APPROVED** [review]

**Pull request merged**